src/FluentValidation.Tests/ValidatorTesterTester.cs

```diff
@@ -86,6 +91,18 @@ public class ValidatorTesterTester {
 			});
 		}
 
+		[Fact]
+		public void ShouldHaveValidationError_Identity_Should_not_throw_when_there_are_validation_errors() {
+			validator.ShouldHaveValidationErrorFor<Person>(x => x, new Person { Forename = "Helle", Surname = "Helle" }, "default");
+		}
+
+		[Fact]
+		public void ShouldHaveValidationError_Identity_Should_throw_when_there_are_no_validation_errors() {

```

Comment:
These tests will also need inverse versions that cause the testhelper to throw an exception (which you'll need to catch and assert against, see `ShouldNotHaveValidationError_should_throw_when_there_are_errors_with_preconstructed_object` as an example)